doc/guides/backporting-to-release-lines.md

```diff
@@ -8,56 +8,54 @@ pad while preparing a release. The branch name is formatted as follows:
 
 ### Active staging branches
 
-| Release Line | Staging Branch |
-| ------------ | -------------- |
-| `v7.x`       | `v7.x-staging` |
-| `v6.x`       | `v6.x-staging` |
-| `v4.x`       | `v4.x-staging` |
+| Release Line | Staging Branch | LTS Status  |
+| ------------ | -------------- | ----------  |
+| `v8.x`       | `v8.x-staging` | Current     |
+| `v7.x`       | `v7.x-staging` | Maintenance |
+| `v6.x`       | `v6.x-staging` | LTS         |
+| `v4.x`       | `v4.x-staging` | Maintenance |
 
 ## What needs to be backported?
 
 If a cherry-pick from master does not land cleanly on a staging branch, the
 releaser will mark the pull request with a particular label for that release
-line, specifying to our tooling that this pull request should not be included.
-The releaser will then add a comment that a backport is needed.
+line (e.g. `backport-requested-v8.x`), specifying to our tooling that this
+pull request should not be included. The releaser will then add a comment
+that a backport is needed.
```

Comment:
`that a backport is needed.` -> `requesting that a backport pull request be made`